mcc/src/client/AnimalRequest/app.tsx

```diff
@@ -1,5 +1,5 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
```

Comment:
for my own edification, what are best practices in typescript for semicolons? if we introduce eslint, presumably that'll enforce some of this?

Reply:
Ideally, no semicolons anywhere. I've been working on enforcing that in my eslint research -- it might be something that we can just run a task to do automatically for us

Reply:
interesting on semicolons. yeah, i think we should try to incorporate some coding standards